### PR TITLE
Entity QC: Zones 027 to 029

### PR DESCRIPTION
**Pull request by wrenffxi**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

* Addendum to zones [027](https://github.com/LandSandBoat/server/pull/110) and [028](https://github.com/LandSandBoat/server/pull/112), in which I mistakenly didn't update mob coords.

In zone 029 (Riverne #B01),

* fix incorrectly ordered Unstable Displacements
* rename qm1, qm2, etc. to something more meaningful
* adjust mob coords, level ranges, subanimations from Nyu's retail caps
* update NPC data from Nyu's retail caps

### CODE REVIEW

**Review by claywar — APPROVED** [review]

File: scripts/zones/Riverne-Site_B01/npcs/Unstable_Displacement.lua [hunk 1]
```diff
@@ -11,21 +13,15 @@ require("scripts/globals/bcnm")
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
-    local offset = npc:getID() - ID.npc.DISPLACEMENT_OFFSET
-    if (offset == 5 and TradeBCNM(player, npc, trade)) then -- The Wyrmking Descends
-        return
-    end
+    TradeBCNM(player, npc, trade)
```
Comment (TeoTwawki) [inline scripts/zones/Riverne-Site_B01/npcs/Unstable_Displacement.lua:16]: sooo we don't need to know _which_ unstable displacement it is before hitting the BCNM code anymore?

Comment (TeoTwawki) [reply scripts/zones/Riverne-Site_B01/npcs/Unstable_Displacement.lua:16]: from discord:
> Re: the unstable displacement change in the PR, looks like there's only one "officially named" unstable displacement in B01 now:
INSERT INTO npc_list VALUES (16896217,'Unstable_Displacement','Unstable Displacement',0,-612.800,1.750,693.190,32769,50,50,0,0,0,0,3,0x0000320000000000000000000000000000000000,0,'COP',0);

Comment (wrenffxi) [reply scripts/zones/Riverne-Site_B01/npcs/Unstable_Displacement.lua:16]: Yup!  This NPC is the only one with name = 'Unstable_Displacement' in the zone.  The other three "Unstable Displacement" NPCs in the zone are `_0t1`, `_0t2`, and `_0t3`.

Comment (TeoTwawki) [reply scripts/zones/Riverne-Site_B01/npcs/Unstable_Displacement.lua:16]: Not sure how long ago it was they changed, but waaay back when they were brand new it was all one giant script for them all lol. In _both_ rivernes (1 script for each zone).

**Pull request merged**